presence:Made it so that stats of even invisible users are been recorded (#32740)
zulip/zulip: help/status-and-availability.md

With this setting, your "Last active" time displayed to other users in
the UI will be frozen as the approximate time you enabled this setting.
Your Zulip usage will still be included in your organization's [statistics](/help/analytics).

> 

Reviewer: Why is there an extra line break?

> !!! warn ""
> 
>     **Note**: Users who have Invisible mode enabled will be included in aggregate usage statistics.

Reviewer: ```suggestion
Your Zulip usage will still be included in your organization's [statistics](/help/analytics).
```
